src/util/pagination.tsx

```diff
@@ -17,18 +18,51 @@ export const paginationOptions = [
   },
 ];
 
-export const usePagination = (
-  data: MainTableRow[]
-): {
+export interface Pagination {
   pageData: MainTableRow[];
   currentPage: number;
   paginate: (pageNumber: number) => void;
   pageSize: number;
   setPageSize: (newSize: number) => void;
   totalPages: number;
-} => {
+  updateSort: (sort: string | null | undefined) => void;
```

Comment:
I think this can be shortened to:

```suggestion
  updateSort: (sort?: string) => void;
```

Reply:
Wouldn't it still be compatible? I tested it locally and it seemed to work.

Reply:
True, with `sort?: string | null` it's already better anyway.